### PR TITLE
Fixes #27236 - Send verbosity level to the smart proxy

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: app/lib/actions/foreman_ansible/helpers/host_common.rb [hunk 1]
```diff
@@ -29,10 +29,6 @@ def fill_continuous_output(continuous_output)
         rescue StandardError => e
           continuous_output.add_exception(_('Error loading data from proxy'), e)
         end
-
-        def find_options
```
Comment (adamruzicka) [inline app/lib/actions/foreman_ansible/helpers/host_common.rb]: The entire module seems like a leftover from when foreman ansible implemented its own actions. A quick grep didn't find it being used anywhere. How about I drop it entirely?

**Comment by adamruzicka** [comment]
[test foreman_ansible]

**Comment by ares** [comment]
[test foreman_ansible] rekick

**Comment by adamruzicka** [comment]
[test foreman_ansible] one more time. _Daft Punk starts playing in the background_

**Comment by ares** [comment]
Thanks @adamruzicka, merging, the failures are caused by templates snapshots.

**Pull request merged**